source/controller/wizard_controller.py

```diff
@@ -166,6 +166,14 @@ def verify_and_save_project_folder(self):
             self.update_complete_project_files_page(complete_project_location=False)
 
     def save_project_description(self):
```

Comment:
Do not forget to add returns and types to the methods:
```suggestion
    def update_project_description(self) -> None:
```

Reply:
Ok, thank you for the help. I will change my errors